### ISSUE TITLE
UI regression on replying message with touch pad

### ISSUE THREAD

**Issue by Jack-Works** [UnigramDev/Unigram#2662]
**Describe the bug**

**To Reproduce**

In this UI,

![image](https://user-images.githubusercontent.com/5390719/154428478-08150c94-125b-422d-bb71-3d76d01f238d.png)

Swipe left use 2 fingers on the high precision touchpad on a message

**Expected behavior**

The message can be swiped left, and when the finger releases, it becomes the "reply" state.

![image](https://user-images.githubusercontent.com/5390719/154428586-5b2e59e4-f0b8-4bea-85d5-e27b181abe30.png)

In the newer version, this useful feature seems removed.

**Version Info**
 - Unigram: 8.5 (6982) x64
 - Windows: 10

**Additional context**

**Comment by makisukurisu** [comment]
Additional note: horizontal scroll causes vertical scroll in chat list and chat window.

**Comment by FrayxRulez** [comment]
I'm using the same version of the app and I can't reproduce the issue.
Additionally, there was no changes to this logic since October: https://github.com/UnigramDev/Unigram/commits/develop/Unigram/Unigram/Controls/Chats/ChatListViewItem.cs

**Comment by makisukurisu** [comment]
You can try to see if SwipeControl works for your machine in [WinUI3 Gallery](https://www.microsoft.com/store/productId/9P3JFPWWDZRC), if it does this issue must be investigated further, if it doesn't - this is caused by driver your trackpad is using (if you don't have one - you might want to see if there is any for your machine and try with it)

Thanks for the report.
P.S. _ELAN 21.2.27.1 doesn't inform system about swipe actions, apparently. Or it does, but in such a way modern apps cannot understand._

**Comment by Jack-Works** [comment]
> You can try to see if SwipeControl works for your machine in [WinUI 3 Controls Gallery](https://www.microsoft.com/store/productId/9P3JFPWWDZRC), if it does this issue must be investigated further

Yes, I can swipe from left to right and reveal the "accept" and the "flag" button in the SwipeView.

When I do the same action on one of the messages in the Unigram, it does not work.

I tried to swipe in the chat list and it works. Looks like it only fails in the chat UI.

**Comment by caeslucio** [comment]
I've using a Windows 10 touchscreen leptop Dell 15r 5537, and I'm unable to use swipe to reply on current version of Unigram, both Store (8.6.7182.0) and Direct (8.7.7352.0) versions.

**Comment by caeslucio** [comment]
> 
On touchscreen I tried to swipe in the chat list and it works here too

**Comment by caeslucio** [comment]
But it working like in the chat UI, because when I swipe to right it shows the reply icon

**Comment by lloydo3000** [comment]
> I tried to swipe in the chat list and it works. Looks like it only fails in the chat UI.

Same for me

**Comment by MarcAnt01** [comment]
@caeslucio @lloydo3000 this is not related, since here the user here is unable both to reply and forward the message with the gestures, while your issue seems to be the chats forwarding menu not loading, while gestures are working properly, please consider opening a separated ticket

**Comment by lloydo3000** [comment]
Thank you but my issued is exactly as described in the ticket. Gestures work in other apps, but since a change in Unigram a few months ago, the do not work on messages in a chat (they do work swiping on the menus, as described above). The gestures don't work either on touchpads or on touchscreens. I have seen this on multiple computers now. I would love a fix as it was a great productivity booster.

**Comment by Jack-Works** [comment]
I just upgraded to Windows 11 22H2, and with Unigram 8.6 (7182) this problem no longer existed. Another of my PC is in Windows 10 21H2 Unigram 8.6 (7182), this problem still exists.

**Comment by FrayxRulez** [comment]
Yeah, late comment here. I believe that the regression was introduced by this commit and I didn't notice because there was a (undocumented) breaking change in Windows 11: https://github.com/UnigramDev/Unigram/commit/2af040b38be04a30be657914709313e7adce6269

**Comment by FrayxRulez** [comment]
Doesn't seem to be fixed actually

**Comment by FrayxRulez** [comment]
Given that this bug only affects Windows 10 and there are less and less users running the app on this OS version, I'll close this as won't fix
